Graphs/Table/index.js

```diff
@@ -103,6 +103,26 @@ const setSelectedRows = (props) => {
     return !isEmpty(selectedRows) ? (selectedRows[requestId] || []) : [];
 }
 
+
+// get columns current displayOption type selected.
+const selectedContext = (props) => {
+    const { context, columns } = props;
+    let key = [];
+    columns.forEach(column => {
+        if(column.displayOption){
+            key.push(...Object.keys(column.displayOption));
+        }
+    });
+    key = uniq(key);
+    let queryKey = 'ALL';
```

Comment:
You have a default of ALL. And at L120 you are appending more keys without delimiter. Can you give an example here? It is difficult to understand the need for this function.

Reply:
Done.